lib/routes/route.js

```diff
@@ -330,11 +338,20 @@ module.exports = function(app, router) {
                 logger.error(`Could not find image of ID ${id} for direct link, serving "removed" placeholder image via redirect`);
                 res.redirect("/images/removed.png");
             } else {
-                res.status(200);
-                res.append("Cache-Control", "public, max-age=2592000");
-                res.type(imageEntry[0].mimetype);
-                res.send(imageEntry[0].data.buffer); // Send the file data to the browser.
-                logger.info("Served image " + imageEntry[0].id + " by direct link");
+                if(req.query.res && req.query.res === "low") {
```

Comment:
Hmm, perhaps we should have two separate options, `res` for setting image quality (see https://sharp.pixelplumbing.com/api-output - note: seems like only jpeg and png images can have their quality changed via this API) and `size` for setting width of the image like what's here now. What do you think?

Reply:
Since the `quality` attribute was only for these couple of formats, I have resized the image in general so that they appear to be of lower quality. 
Otherwise, we could convert the other format of images to png and then reduce the quality?
Or, for png and jpeg images, we could set `res` option while for other images we could set `size` option as query parameter.